lib/windows/winlean.nim

```diff
@@ -409,10 +409,10 @@ else:
     importc: "GetCommandLineA", stdcall, dynlib: "kernel32".}
 
 proc rdFileTime*(f: FILETIME): int64 =
-  result = ze64(f.dwLowDateTime) or (ze64(f.dwHighDateTime) shl 32)
+  result = f.dwLowDateTime.int64 or (f.dwHighDateTime.int64 shl 32)
 
 proc rdFileSize*(f: WIN32_FIND_DATA): int64 =
-  result = ze64(f.nFileSizeLow) or (ze64(f.nFileSizeHigh) shl 32)
+  result = f.nFileSizeLow.int64 or (f.nFileSizeHigh.int64 shl 32)
```

Comment:
`rdFileSize()` return type must be `uint64` and conversion of `nFileSizeLow` and `nFileSizeHigh` must be also `uint64`.

Reply:
Sure thing, I stopped halfway because I'm no Windows user and the whole change-push-wait_for_CI loop was taking too long.